tests/unit/test_util.py

```diff
@@ -0,0 +1,41 @@
+"""
+Test the utility methods
+"""
+
+import pytest  # NOQA
+from koji_wrapper.util import convert_to_list
+
+
+class TestConvertToList(object):
```

Comment:
I would include a test that includes an unexpected data type to make sure we hard handling things correctly.

Reply:
Also a good idea, thanks for the feedback!